test/test_linalg.py

```diff
@@ -5447,7 +5447,6 @@ def run_subtest(matrix_size, batches, device, pivot, singular=False, a=None):
                 self.assertEqual(a_LU, a_LU_info)
                 self.assertEqual(pivots_info, pivots)
 
-
```

Comment:
This PR doesn't need to touch this file.
Please use `git checkout master -- aten/src/ATen/native/cuda/ScatterGatherKernel.cu` to undo changes to the file.